### ISSUE TITLE
Requests

### ISSUE THREAD

**Issue by Pinkfag** [chartanon/moechart#2]
Could you make a SVG for a circle with a question mark in it? What tool were you using to make them in the first place?

**Comment by Pinkfag** [comment]
@chartanon in case you didn't get a notification

**Comment by chartanon** [comment]
Here: ![icon-question-mark](https://github.com/chartanon/moechart/assets/132846777/5d071a86-f79c-4ff9-a42e-8a3a54d817f4)
I made the icons with inkscape

**Comment by Pinkfag** [comment]
Cool, thanks! do we want to try hosting now?

**Comment by Pinkfag** [comment]
@chartanon One more request, could you make a clock SVG? Planning to use this to sort chronologically by release date

**Comment by chartanon** [comment]
Clock: ![icon-clock](https://github.com/chartanon/moechart/assets/132846777/aaf66e38-e8a6-4919-8599-76b323b9a13e)
Edit: Maybe this looks better ![icon-clock2](https://github.com/chartanon/moechart/assets/132846777/b73cc1df-9931-490f-9441-66bb09b16e6b)

**Comment by Pinkfag** [comment]
@chartanon thoughts on adding a "babby's first" or "/vn/ recommended" filter for beginners? We could talk about which ones we think would be appropriate, or maybe we could do a poll on /vn/ about it. And then just add it as an attribute tag to a couple VNs.

My suggestion is we do a face with a single hair + pacifier, or maybe just a thumbs up or something as the icon.

Also thanks for the clock!

**Comment by Pinkfag** [comment]
Also if we're doing a recommended section, we should write up some short descriptions as to what each beginner friendly VN entails and why it's a good first read

**Comment by Pinkfag** [comment]
though threads have been kind of unusable lately with all the shitposts and trolling, maybe we should just work on this without their help

**Comment by chartanon** [comment]
> thoughts on adding a "babby's first" or "/vn/ recommended" filter for beginners?

I'm all for it. I'd go with a thumbs up icon, or maybe a star?

**Comment by Pinkfag** [comment]
Up to you since you'll be making the icon. Maybe some sort of stamp? 
![image](https://github.com/chartanon/moechart/assets/87510981/734887cc-1061-45e8-a131-994763883d21)

**Comment by Pinkfag** [comment]
@chartanon another icon request, can we get a magnifying glass? Planning on using this as the icon for a text search tool

Scratch this idea, I don't think there's any reason a user should be searching for a specific VN in the moechart, since we're not really providing any important data here anyways. Whatever they're looking for should be checked on vndb.

**Comment by Pinkfag** [comment]
Also, thoughts on taking out the FD label and adding the FDs/sequels to the chart? Do you think that might bloat the list too much? Maybe we could replace it with a "part of a series" tag? I dunno, just spitballing. Maybe it's a bad idea

**Comment by Pinkfag** [comment]
One more thing. This is completely excessive, but I was thinking maybe we could get a dice icon and make a 'randomized suggestion' button.

**Comment by Pinkfag** [comment]
New request: a music note icon, please!

**Comment by chartanon** [comment]
Some icons (I just made a star for recommended VNs)
![icon-star](https://github.com/chartanon/moechart/assets/132846777/929eac40-9447-4f9f-85e7-0173d7925035)
![icon-dice](https://github.com/chartanon/moechart/assets/132846777/358d0a5b-7a53-4c2f-87d5-926da9575729)
![icon-note](https://github.com/chartanon/moechart/assets/132846777/fd32f08d-6514-4f8f-9005-da32e088f049)


> Also, thoughts on taking out the FD label and adding the FDs/sequels to the chart? Do you think that might bloat the list too much?

I think it would be nice to have FDs and sequels directly on the chart. If people sort chronologically they probably expect them to show up as new releases.
But it's also true that they bloat the chart quite a bit. I'm not sure what to do about. I guess the most straightforward solution would be adding a way to easily hide VNs with certain attributes

**Comment by Pinkfag** [comment]
Yeah I was thinking of having the filters have a second state where when you clicked the buttons twice, they'd exclude VNs instead of include them. But the logic for doing that is a massive pain. I think we have a few options here:

1. We add FDs to the chart. We remove the icon. We add a new filter that hides FDs, maybe have it turned on by default.
1. We add some sort of secondary UI, maybe clicking a specific part of the VN on the chart will display it. We display the FDs here. (Think of it like card games like hearthstone where you can right click a card and it'll show all the tokens and what not that are tied to it).
1. Do nothing, since the FD icon exists and clicking on a VN takes them to VNDB which shows the FDs right there anyways.

What are your thoughts? Do you think there might be another solution? Also, what do you think should belong in the recommended VNs list?

**Comment by Pinkfag** [comment]
#1 is a very easy solution for me to add, #2 is obviously more work. However I do think that if we go with a secondary UI where we focus the whole page on the VN, it'll probably look very empty with the existing data we have for each entry, so chances are we might need to fill in the data with more info.

**Comment by Pinkfag** [comment]
Requesting a down arrow icon, but maybe add some sort of outline/extra stuff to make it look like there's motion involved. I'm planning on using this as a "scroll down to upcoming section"

**Comment by Pinkfag** [comment]
@chartanon do you know when I'm sending messages in this thread without me pinging you?

**Comment by chartanon** [comment]
Yes, I always get notifications. I was just busy over the weekend.

>1. We add FDs to the chart. We remove the icon. We add a new filter that hides FDs, maybe have it turned on by default.

This is probably the best solution. We could just add the FD icon directly to the FDs and update the legend accordingly. The icon now means "This VN is a fandisc/sequel". That way people can still identify FDs if the filter is off.
Another problem are VNs with a lot of FDs like Majikoi (I'm not even sure if that should be on the chart, tbh) and these single heroine VNs with 4 or more VNs in a series. But I guess it will be fine with the filter.

I'll probably open another issue where we collect recommended VNs. Maybe another issue where we discuss removals/additions would be a good idea. For now I mostly copied the VNs from old chart without thinking too much

**Comment by chartanon** [comment]
Scroll arrow: ![icon-scroll](https://github.com/chartanon/moechart/assets/132846777/37ca0a59-7a6c-4deb-9c88-ec6b62e9ec63)
I'm not sure if that's what you had in mind

**Comment by Pinkfag** [comment]
Scroll arrow is perfect, thanks!

**Comment by Pinkfag** [comment]
When you have time, please add all the moege FD data to my current branch, `additional-query-tools`

**Comment by Pinkfag** [comment]
Scratch that, I'll make a new branch for you to do that. Going to merge my changes now

**Comment by Pinkfag** [comment]
Branch is called [feature/split-fandiscs-to-main-chart](https://github.com/chartanon/moechart/tree/feature/split-fandiscs-to-main-chart)

**Comment by chartanon** [comment]
I'm on it

**Comment by Pinkfag** [comment]
Thanks, also when are you planning on adding the new shiravune vn?

**Comment by Pinkfag** [comment]
Oh yeah I forgot to mention, please tag all the FDs with the current FD icon. I'll just use that for the hiding filter

**Comment by Pinkfag** [comment]
Not sure if it was you in the thread, but there's also talks about 9nine not being in the moechart. Should those be added?

**Comment by chartanon** [comment]
We can add it and see how many people complain, Tamayura Mirai too. I will get to them after adding all the FDs and sequels

**Comment by Pinkfag** [comment]
Sounds good, take your time

**Comment by chartanon** [comment]
Everything should be on the chart now. I combined some series like Majikoi A-1-A4 or the Whirpool games into a single entry to get rid of the worst bloat. I'll probably do the same with all the qureate shovelware that's polluting the chart, but not right now.

**Comment by chartanon** [comment]
Oh and I noticed that a lot of FDs have scenario/heroine selection. I was thinking about adding an icon for that down the line. I only marked them with comments for now and left it as branching plot.
Also I only commented the old HAS_SEQUELS attribute out in case we want to reintroduce it. Maybe only show it if the FDs are hidden or something? It would need a different icon though. It's just an idea, I'm not against removing it completely

**Comment by Pinkfag** [comment]
Can you make a close icon SVG? An X is fine, but if there's anything more fancy you'd like to do that'd be cool too

**Comment by Pinkfag** [comment]
Also a bookmark icon would be nice

**Comment by chartanon** [comment]
Icons
Close ![icon-close](https://github.com/chartanon/moechart/assets/132846777/fee5d108-7982-41c4-8087-6cdd500a2715)
Bookmark ![icon-bookmark](https://github.com/chartanon/moechart/assets/132846777/58022ddf-21fd-4b21-bbff-5a6961ecdd02)

Also what do you think about doing the cards like this?
![screenshot](https://github.com/chartanon/moechart/assets/132846777/5d021a3c-4147-45e7-9c69-9262977ca38c)

**Comment by Pinkfag** [comment]
Without the color you mean? Yeah sure we can remove it. Scaling the icon/text container's distance is doable, but it's kind of a pain to implement

**Comment by chartanon** [comment]
Not quite. I only did a quick experiment with the element inspector so I don't know how hard it is to do in general but basically I reduced the width and height of the thumbnails by offset * number_of_sequels. That way the stack will always have the same size and no further calculations are needed except for the margin of the IconsContainer which has to be increased by offset * number_of_sequels.

For the icons I think we can just use a css filter to get the glow effect
